ipn/ipnlocal/local.go

```diff
@@ -1303,6 +1303,14 @@ func (b *LocalBackend) updateFilterLocked(netMap *netmap.NetworkMap, prefs ipn.P
 			localNetsB.AddPrefix(p)
 		}
 		packetFilter = netMap.PacketFilter
+
+		if packetFilterPermitsUnlockedNodes(netMap.Peers, packetFilter) {
+			err := errors.New("server sent invalid packet filter permitting traffic to unlocked nodes; rejecting all packets for safety")
```

Comment:
Should we reference the node name or ID in the error so we have a thread to pull when debugging?

Reply:
I'd rather keep it simple in the client. If we ever see this, we can run this same code as needed server-side but fleshed out to include debug info.